### PR TITLE
Speakers list page

### PR DESCRIPTION
**Pull request by filipeximenes**
- Adds speakers list page to the navbar
- List all speakers (main and additional) related to approved proposals
- Missing tests: https://github.com/pybay/pybay/issues/72
- Needs some work on CSS: https://github.com/pybay/pybay/issues/73

### CODE REVIEW

File: pybay/views.py [hunk 1]
```diff
@@ -44,3 +45,19 @@ def pybay_cfp_create(request):
...
         form = CallForProposalForm()
         return render(request, 'frontend/cfp.html', {'form': form})
+
+
+def pybay_speakers_list(request):
+    accepted_proposals = Proposal.objects.filter(
+        result__status='accepted')
+    speakers = []
+    for proposal in accepted_proposals:
+        speakers += list(proposal.speakers())
```
Comment (PirosB3) [inline pybay/views.py:55]: This can lead to duplicate speakers

Comment (PirosB3) [reply pybay/views.py:55]: oh sorry! didn't see above :)

**Comment by PirosB3** [comment]
brilliant! I will test today and merge

**Comment by sfpython** [comment]
Excellent!   Thank you so much @filipeximenes.  @PirosB3, I'll let you do the merging.

**Pull request merged**